packages/unenv-preset/tests/index.test.ts

```diff
@@ -0,0 +1,62 @@
+import fs from "node:fs";
+import path from "node:path";
+import { platform } from "node:process";
+import { fileURLToPath } from "node:url";
+import { fetch } from "undici";
+import { afterAll, beforeAll, describe, expect, test } from "vitest";
+import { runWranglerDev } from "../../../fixtures/shared/src/run-wrangler-long-lived";
+import { TESTS } from "./worker/index";
+
+const pkgDir = path.resolve(fileURLToPath(import.meta.url), "../..");
+// workerd binary
+const workerdBin = platform === "win32" ? "workerd.ps1" : "workerd";
```

Comment:
Perhaps .cmd instead?

```suggestion
const workerdBin = platform === "win32" ? "workerd.cmd" : "workerd";
```

Not sure if that is why the test is timing out or not...

Reply:
We have 0 test in this repo with either .cmd or .ps1.

I'll try .cmd once and if it doesn't work, I'll skipIf(win32). Does that sound good?